### PR TITLE
[SPARK-12692][BUILD][GRAPHX] Scala style: Fix the style violation (Space before "," or ":")

### PR DESCRIPTION
**Pull request by sarutak**
Fix the style violation (space before `,` and `:`).
This PR is a followup for #10643.

### CODE REVIEW

File: graphx/src/main/scala/org/apache/spark/graphx/impl/ReplicatedVertexView.scala [hunk 1]
```diff
@@ -41,8 +41,8 @@ class ReplicatedVertexView[VD: ClassTag, ED: ClassTag](
    * shipping level.
    */
   def withEdges[VD2: ClassTag, ED2: ClassTag](
-      edges_ : EdgeRDDImpl[ED2, VD2]): ReplicatedVertexView[VD2, ED2] = {
-    new ReplicatedVertexView(edges_, hasSrcId, hasDstId)
+      _edges: EdgeRDDImpl[ED2, VD2]): ReplicatedVertexView[VD2, ED2] = {
```
Comment (sarutak) [inline graphx/src/main/scala/org/apache/spark/graphx/impl/ReplicatedVertexView.scala]: I noticed we need a white space if the name of variables ends with "_".
Those variables might be used internally so can we rename to which like prepended `_` right?

**Comment by sarutak** [comment]
retest this please.

**Comment by sarutak** [comment]
MiMa said

```
[error] (streaming-mqtt/*:mimaPreviousClassfiles) sbt.ResolveException: unresolved dependency: org.eclipse.paho#org.eclipse.paho.client.mqttv3;1.0.1: not found
```
But It's funny. I think the dependency of streaming-mqtt is not related to this change.

**Comment by sarutak** [comment]
retest this please.

File: graphx/src/main/scala/org/apache/spark/graphx/impl/ReplicatedVertexView.scala (hunk 1 above)
Comment (rxin) [inline graphx/src/main/scala/org/apache/spark/graphx/impl/ReplicatedVertexView.scala]: yes

**Comment by rxin** [comment]
LGTM

**Comment by sarutak** [comment]
retest this please.

**Comment by rxin** [comment]
I've merged this.

**Pull request closed without merging**